gulpfile.js

```diff
@@ -13,7 +13,7 @@ gulp.task('copy', function() {
 
 gulp.task('babel', function() {
   return browserify({entries: 'js/app.jsx', extensions: ['.jsx'], debug: true})
-    .transform('babelify', {presets: ['es2015', 'react']})
+    .transform('babelify', {presets: ['env', 'react']})
```

Comment:
is this a change in api from the upgrades?

Reply:
yeah "env" is intended to keep up with each ES release, so one only needs to upgrade a dependency rather than chaining successive releases (e.g. `['es2015', 'es2016', 'es2017', 'react']`). See http://babeljs.io/env for details.